.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,25 +1,47 @@
 ---
 name: Bug Report
 about: Something doesn't work correctly in Atmosphère.
-
+#assignees:
 ---
 
 ## Bug Report
 
+[ If any section does not apply, use nil. ]  
+[ Lines between [ ] (square brackets) should be removed before posting. ]
+
 ### What's the issue you encountered?
 
+[ Describe the issue in detail and what you were doing beforehand. ]  
+[ Did you make any changes related to Atmosphère itself? ]  
+[ If so make sure to include details relating to what exactly you changed. ]
+
 ### How can the issue be reproduced?
 
-### Crash report?
+[ Include a detailed step by step process for recreating your issue. ]
+[ Note: If software you have installed can be used to make 'backups' of game files or distributed explicitly for piracy and are connected to why Atmosphère crashed, we will not provide support related to this issue. ]
```

Comment:
This sentence is very hard to parse. It can definitely be cleaned up.

Reply:
Fixed with new template for support disclaimers at the top of bug reports by default.